src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationsMappingService.kt

```diff
@@ -9,48 +9,43 @@ import org.springframework.web.reactive.function.client.WebClientResponseExcepti
 import reactor.core.publisher.Mono
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.integration.history.CreateMappingResult
 import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.integration.history.MigrationMapping
-import java.time.LocalDateTime
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.nomismappings.model.AdjudicationAllMappingDto
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.nomismappings.model.AdjudicationMappingDto
 
 @Service
 class AdjudicationsMappingService(@Qualifier("mappingApiWebClient") webClient: WebClient) :
-  MigrationMapping<AdjudicationMapping>(domainUrl = "/mapping/adjudications", webClient) {
+  MigrationMapping<AdjudicationAllMappingDto>(domainUrl = "/mapping/adjudications", webClient) {
 
-  suspend fun findNomisMapping(adjudicationNumber: Long, chargeSequence: Int): AdjudicationMapping? {
+  override fun createMappingUrl(): String {
```

Comment:
base class assumed the GET and POST urls were the same - in our case they are different,. We read the mapping for the main adjudication but write mapping for all child entities